scripts/prepare_conv_pp.py

```diff
@@ -160,4 +160,4 @@ def set_unit(df, unit_dict):
     conv_scalars_prepared.replace(carrier_dict, inplace=True)
 
     # export prepared conventional power plant data
-    conv_scalars_prepared.to_csv(out_path, index=False)
+    conv_scalars_prepared.to_csv(out_path, sep=";", index=False)
```

Comment:
Better use `save_df` for all these instances where you save some data in oemof-b3 resources format.

Reply:
I cannot recall why it seemed necessary to ignore the index when saving. Could be obsolete.

Reply:
I wanted to, but that does not work in here because of `index=False`. `save_df` uses `index=True` as default and does not have an input parameter `index` so far, where I could change this.

Reply:
You're right. It's obsolete. I tested this with `dp.save_df` in `prepare_conv_pp.py`, `snakemake -j1 prepare_conv_pp` and `snakemake -j1 results/_resources/plots/scal_conv_pp_var_capacity_net_el.png`.

Reply:
Adjusted with commit https://github.com/rl-institut/oemof-B3/pull/110/commits/c3b8f130069303ed5a04e62a30178a0de857f48d.